.cirrus.yml

```diff
@@ -98,130 +98,82 @@ linux_mac_task:
-    # ^  deactivate SSL checking to avoid error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:661)
-  pip_cache:
-    folder: '%PIP_CACHE%'
-  pre_commit_cache:
-    folder: '%PRE_COMMIT_HOME%'
+    CIRRUS_SHELL: bash
+    PATH: /c/Python39:/c/Python39/Scripts:/c/tools:${PATH}
   install_script:
     # Activate long file paths to avoid some errors
-    - REG ADD "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\FileSystem" /v LongPathsEnabled /t REG_DWORD /d 1 /f
-    # Set Windows encoding to UTF-8
-    - REG ADD "HKEY_CURRENT_USER\Software\Microsoft\Command Processor" /v Autorun /t REG_SZ /d "@chcp 65001>nul" /f
-    - python -m ensurepip
-    - python -m pip install --upgrade --user pip setuptools certifi tox pre-commit
-  prepare_script: *prepare
-  clean_workspace_script:
-    # Avoid information carried from one run to the other
-    # CMD is not capable of globbing, so we have to use PowerShell
-    - ps: (rm -Recurse -Force -ErrorAction SilentlyContinue .tox,junit-*.xml)
-  build_script: *build
-  test_script: *test
-  always: *upload-junit
-
-
-coverage_task:
-  name: coverage (Linux - 3.8)
-  clone_script: *clone
-  container: {image: "python:3.8-bullseye"}
-  env:
-    COVERAGE: yes
-  depends_on:
-    - test (Linux - 3.7)
-    - test (Linux - 3.8)
-    - test (Linux - 3.9)
-    - test (Linux - 3.10)
-    - test (Linux - Anaconda)
-    - test (OS X)
-  pip_install_script:
-    pip install --user --upgrade coverage coveralls pre-commit
-  precommit_script:
-    - pre-commit install
-    - pre-commit run --all-files
-  <<: *REGULAR_TASK_TEMPLATE
-  coverage_script:
-    - coveralls
+    - ps: New-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem" -Name "LongPathsEnabled" -Value 1 -PropertyType DWORD -Force
+    - choco install -y --no-progress python3 --version=3.9.10 --params "/NoLockdown"
+    - choco install -y --no-progress vim
+    - pip install --upgrade certifi
+    - python -m pip install -U pip tox pipx
+  <<: *test-template
 
+finalize_task:
+  container: {image: "python:3.10-bullseye"}
+  depends_on: [test]
+  <<: *task-template
+  install_script: pip install coveralls
+  finalize_coverage_script: coveralls --finish
 
 linkcheck_task:
-  name: linkcheck (Linux - 3.8)
-  # only_if: $CIRRUS_BRANCH == 'master'
+  name: linkcheck (Linux - 3.10)
+  container: {image: "python:3.10-bullseye"}
+  depends_on: [finalize]
   allow_failures: true
-  clone_script: *clone
-  container: {image: "python:3.8-bullseye"}
-  depends_on:
-    - test (Linux - 3.8)
-  pip_cache: *pip-cache
-  tox_install_script:
-    - python -m pip install --upgrade pip setuptools tox
-  prepare_script: *prepare
-  linkcheck_script:
-    - python -m tox -e linkcheck -- -q
+  dist_cache: {folder: dist, fingerprint_script: echo $CIRRUS_BUILD_ID}  # download
+  <<: *task-template
+  install_script: pip install tox
+  linkcheck_script: tox --installpkg dist/*.whl -e linkcheck -- -q
+
+# publish_task:
```

Comment:
Here we can decide to uncomment this task, but then we should also remove the [`.github/workflows/publish-package.yml`](https://github.com/pyscaffold/pyscaffold/blob/master/.github/workflows/publish-package.yml) file otherwise we will end up trying to publish the package twice (which will result in error).

Reply:
Yes, should we add a comment about this remark? So that we don't run into this problem in the future?